core/src/main/resources/static/js/synchronizeAcls.js

```diff
@@ -214,6 +214,7 @@ app.controller("synchronizeAclsCtrl", function($scope, $http, $location, $window
             serviceInput['aclSsl'] = acl_ssl;
             serviceInput['aclType'] = acltype;
             serviceInput['envSelected'] = $scope.getAcls.envName;
+            serviceInput['aclId'] = aclId;
 
```

Comment:
will this be null or undefined when syncing from a non aiven system?

Reply:
from FE 'undefined' would be sent, and gets translated to null in java. Tested locally.